docs/node/run-your-node/keymanager-node.md

```diff
@@ -0,0 +1,153 @@
+# Key manager Node
+
+:::info
+
+These instructions are for setting up a _key manager_ node. Key manager nodes run a special runtime that provides confidentiality to other ParaTimes. If you want to run a _validator_ node instead, see the [instructions for running a validator node](validator-node/README.md). Similarly, if you want to run a _ParaTime_ node instead, see the [instructions for running a ParaTime node](paratime-node.mdx).
+
+:::
+
+:::tip
+
+At time of writing the key manager runtime is deployed only on the Testnet and limited to be run by trustworthy partners.
```

Comment:
Hm, this is true for Cipher and Sapphire, but anyone can run key managers for other ParaTimes. So maybe explicitly mention the two ParaTimes for which this is the case.

Reply:
Mentioned.